src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const populationElements = [...document.querySelectorAll('.population')];
+
+const populations = populationElements.map((country) => {
+  return +country.innerText.split(',').join('');
+});
+
+const totalPopulation = populations.reduce((acc, country) => acc + country, 0);
+
+const totalElement = document.querySelector('.total-population');
+
+totalElement.innerText = totalPopulation.toLocaleString('en-US');
+
+const averageElement = document.querySelector('.average-population');
+
+averageElement.innerText = Math.floor(
+  totalPopulation / populations.length,
+).toLocaleString('en-US');
```

Comment:
The `Math.floor` function is applied incorrectly here. It should wrap the entire division operation to ensure the result is floored before converting to a locale string. Consider changing the code to `Math.floor(totalPopulation / populations.length).toLocaleString('en-US');`.